### PR TITLE
Created tests for EcoNewsDtoRequestValidator

### PR DESCRIPTION
**Pull request by Andrii-Danylenko**
Created tests for EcoNewsDtoRequestValidator

### CODE REVIEW

**Review by SezamUA — COMMENTED** [review]
Everything is right, but It'll be more correct not to use the static value 3 of ValidationConstants.MAX_AMOUNT_OF_TAGS, because if someone changes it - the test won't work correctly. You can form List of tags using "for" statement, for example, with ValidationConstants.MAX_AMOUNT_OF_TAGS + 1 items count.

**Comment by Andrii-Danylenko** [comment]
> Everything is right, but It'll be more correct not to use the static value 3 of ValidationConstants.MAX_AMOUNT_OF_TAGS, because if someone changes it - the test won't work correctly. You can form List of tags using "for" statement, for example, with ValidationConstants.MAX_AMOUNT_OF_TAGS + 1 items count.

Excellent point. I'll fix it in the next commit.

**Review by SezamUA — APPROVED** [review]

**Review by StannyB9 — APPROVED** [review]
Everything is fine. Great job!.

**Review by DZotsenko — APPROVED** [review]

File: core/src/test/java/greencity/validator/EcoNewsDtoRequestValidatorSecondaryTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import java.util.Collections;
+import java.util.stream.Stream;
+
+import static greencity.constant.ServiceValidationConstants.MAX_AMOUNT_OF_TAGS;
+
+@ExtendWith(MockitoExtension.class)
+public class EcoNewsDtoRequestValidatorSecondaryTest {
+    @InjectMocks
+    EcoNewsDtoRequestValidator validator;
```
Comment (maks741) [inline core/src/test/java/greencity/validator/EcoNewsDtoRequestValidatorSecondaryTest.java]:
```suggestion
    private EcoNewsDtoRequestValidator validator;
```

File: core/src/test/java/greencity/validator/EcoNewsDtoRequestValidatorSecondaryTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    AddEcoNewsDtoRequest addEcoNewsDtoRequest;
```
Comment (maks741) [inline core/src/test/java/greencity/validator/EcoNewsDtoRequestValidatorSecondaryTest.java]:
```suggestion
    private AddEcoNewsDtoRequest addEcoNewsDtoRequest;
```

**Review by maks741 — APPROVED** [review]

**Pull request merged**